packages/gatsby-theme-carbon/gatsby-config.js

```diff
@@ -19,6 +19,7 @@ module.exports = themeOptions => {
     lunrOptions = defaultLunrOptions,
     repository,
     mediumAccount = 'carbondesign',
+    gatsbyRemarkPlugins,
```

Comment:
Maybe the default assignment here to follow the other default theme options.

```suggestion
    gatsbyRemarkPlugins = [],
```

Reply:
Done